forcetree.c

```diff
@@ -253,18 +259,12 @@ insert_internal_node(int parent, int subnode, int p_child, int p_toplace,
         /* The new leaf will replace p_child in the parent (done before return)
          * Re-attach that particle to the new leaf.*/
         child_subnode = get_subnode(nfreep, p_child);
+        new_subnode = get_subnode(nfreep, p_toplace);
 
         Father[p_child] = ninsert;
-        force_set_next_node(p_child, nfreep->u.suns[child_subnode], firstnode, lastnode);
```

Comment:
I kept it like this to indicate this is a prepend operation.

Reply:
I found it a bit confusing; maybe a comment is better?